pypgstac/pypgstac/load.py

```diff
@@ -544,8 +544,8 @@ def format_item(self, _item: Union[Path, str, dict]) -> dict:
 
         out["partition"] = partition
 
-        bbox = item.pop("bbox")
-        geojson = item.pop("geometry")
+        bbox = item.get("bbox")
```

Comment:
Is `bbox` used? If not, this line can just be removed

Reply:
Actually I see where it's used now - but I'm not sure I follow the logic. Wouldn't `geojson is None and bbox is not None` fail if the bbox check caused the statement to evaluate to False?

Reply:
Discussed in call, this is being removed and the logic below being changed